cmake/Googletest.cmake

```diff
@@ -8,6 +8,8 @@ macro(fetch_googletest)
         if (NOT GTEST_ROOT)
             set(GTEST_ROOT "${GOOGLETEST_BUILD_ROOT}/googletest-install")
         endif()
+        find_package(GTest QUIET)
+        # At this point GTEST_FOUND is set to True in Release but False in Debug.
     endif()
```

Comment:
Put back the call to `find_package(GTest QUIET)` .

At some point we should switch to using `FetchContent` module introduce in CMake 3.11 but for the sake of time, let's go with the old way of getting this package.

https://cmake.org/cmake/help/v3.11/module/FetchContent.html